app/controllers/admin_controller.rb

```diff
@@ -39,6 +42,14 @@ def users
     # Filter by search text
     users = filter(users, search)
 
+    # Filter by status text
+    case status
```

Comment:
I'm thinking we might want to name this param as `locked` or something more specific than `status` to avoid ambiguity/confusion in the future with other fields that might constitute/relate to a status of some sort. An example of that would be the future request of wanting a `disabled` filter on top of the `locked` filter. Maybe these can be represented as booleans where locked users are displayed when `locked = true`, unlocked users are displayed when `locked = false`, and all users can be represented by `locked = nil`

Reply:
This is a good point @timwongj, I went back and forth on this.  I ended up modeling it more off what we do in the Patient Table and used the column header for locked/unlocked which is status.  If we implement a `disabled` option in the future it might make sense to keep it this way but honestly I could go either way on this.  @holmesie what do you think?

Reply:
I do think `locked` is probably the right way to go here, though I'm not really strongly against `status`. Let's do `locked` just for the sake of making it clearer to someone new looking at this code.

Reply:
@holmesie so do you think what tim suggested is best? have locked be a boolean and if it's nil show them all?

Reply:
done here: https://github.com/SaraAlert/SaraAlert/pull/756/commits/880b8305535fbbb847be27d62ebe02cb77d0988f